src/i18n/locale/en.json

```diff
@@ -360,38 +360,38 @@
       "label": {
         "build_time": "Build time",
         "discord": "Discord",
-        "github": "Github",
+        "github": "GitHub",
         "server": "Server",
-        "server_address": "Server Address",
+        "server_address": "Server address",
         "server_version": "Server version"
       },
       "title": "About"
     },
     "backup": {
       "label": {
-        "backup_restore_failed": "Backup restore failed!",
-        "create_backup": "Create Backup",
-        "create_backup_info": "Backup library as a Tachiyomi backup",
+        "backup_restore_failed": "Could not restore backup",
+        "create_backup": "Create backup",
+        "create_backup_info": "Back up library as a Tachiyomi backup",
         "legacy_backup_unsupported": "legacy backups are not supported!",
         "restore_backup": "Restore Backup",
-        "restore_backup_info": "You can also drag and drop the backup file here to restore",
-        "restored_backup": "Backup restore finished!",
-        "restoring_backup": "Restoring backup..."
+        "restore_backup_info": "You can also drag and drop the backup file here to restore it",
+        "restored_backup": "Backup restored.",
+        "restoring_backup": "Restoring backup…"
       },
       "title": "Backup"
     },
     "label": {
-      "dark_theme": "Dark Theme",
-      "image_cache": "Use image cache",
-      "image_cache_description": "Disabling image cache makes images load faster if you have a slow disk, but uses it much more internet traffic in turn",
-      "manga_item_width": "Manga Item width",
+      "dark_theme": "Dark theme",
+      "image_cache": "Preload images",
```

Comment:
This is still a mess.
What does it actually do, in the active case?

Reply:
in the active case the server will use the cached images (saved in a temp folder), so it will only load a image from the source in case it wasn't loaded previously or the cached image was deleted

Reply:
tbh I don't like the new change as well 😆 
maybe as the title just "Image cache" and as the description something like "Reduces data consumption and loads images faster"?